Octokit/Clients/IIssuesEventsClient.cs

```diff
@@ -21,7 +21,7 @@ public interface IIssuesEventsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The issue number</param>
-        Task<IReadOnlyList<EventInfo>> GetAllForIssue(string owner, string name, int number);
+        Task<IReadOnlyList<EventInfo>> GetAllForIssue(string owner, string name, long number);
```

Comment:
Same thing for this file - `number` here is the issue number itself, no the `EventInfo` `number` that is being deserialized.